route/build/src/histVars_data.f90

```diff
@@ -17,6 +17,20 @@ MODULE histVars_data
   USE globalData,        ONLY: meta_rflx, meta_hflx
   USE globalData,        ONLY: idxSUM,idxIRF,idxKWT, &
                                idxKW,idxMC,idxDW
+  USE globalData,        ONLY: nRch, nRch_mainstem, nRch_trib
+  USE globalData,        ONLY: nHRU, nHRU_mainstem, nHRU_trib
+  USE globalData,        ONLY: nTribOutlet
+  USE globalData,        ONLY: hru_per_proc        ! number of hrus assigned to each proc (size = num of procs
+  USE globalData,        ONLY: rch_per_proc        ! number of reaches assigned to each proc (size = num of procs)
+  ! pio stuff
+  USE globalData,        ONLY: pid, nNodes
+  USE globalData,        ONLY: masterproc
+  USE globalData,        ONLY: mpicom_route
+  USE globalData,        ONLY: pio_typename
+  USE globalData,        ONLY: pioSystem
+  USE nr_utils,          ONLY: arth
+  USE ncio_utils,        ONLY: get_nc
+  USE pio_utils
```

Comment:
pio routines probably start with "pio_" but it's still good to list them explicitly so that their source can be easily tracked down. You do have pio_typename and pioSystem above which could be confused with pio_utils.

Reply:
All the shared procedures in pio_utils module are listed now